sale_product_multi_add/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Cédric Pigeon, ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Sale Product Multi Add",
+    'summary': """
+        Sale Product Multi Add """,
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```

Comment:
@hveficent  URL should be `https://github.com/OCA/sale-workflow`

Reply:
Done